apps/angular/src/app/features/anime/pages/anime/anime-page.component.ts

```diff
@@ -0,0 +1,56 @@
+import { ChangeDetectionStrategy, Component, DestroyRef, inject } from '@angular/core';
+import { CommonModule, AsyncPipe } from '@angular/common';
+import { MatTableModule } from '@angular/material/table';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { BehaviorSubject, Observable, tap, delay } from 'rxjs';
+
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+import { Anime } from '@js-camp/core/models/anime/anime';
+import { Pagination } from '@js-camp/core/models/pagintation';
+
+/** Anime page component. */
+@Component({
+	selector: 'camp-anime-page',
+	standalone: true,
+	imports: [CommonModule, AsyncPipe, MatTableModule, MatProgressSpinnerModule],
+	templateUrl: './anime-page.component.html',
+	styleUrl: './anime-page.component.css',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AnimePageComponent {
+	/** Anime page. */
+	protected readonly animes$: Observable<Pagination<Anime>>;
+
+	/** Loading state. */
+	protected readonly isLoading$ = new BehaviorSubject(false);
+
+	private readonly animeService = inject(AnimeService);
+
+	private readonly destroyRef = inject(DestroyRef);
+
+	/** Columns. */
+	public readonly displayedColumns: string[] = [
+		'image',
+		'titleEnglish',
+		'titleJapanese',
+		'airedStart',
+		'type',
+		'status',
+	];
+
+	public constructor() {
+		this.animes$ = this.createAnimeStream();
+	}
+
+	private createAnimeStream(): Observable<Pagination<Anime>> {
+		return this.animeService.getAnimeList().pipe(
+			tap(() => this.isLoading$.next(true)),
+			takeUntilDestroyed(this.destroyRef),
+		)
+			.pipe(
+				delay(2000),
+				tap(() => this.isLoading$.next(false)),
```

Comment:
You don't have to fix anything here, but try not to overuse the Tap operator, side effects can seriously confuse the logic for analysis. If you can implement the Tap logic in the form of transformation operators such as SwitchMap, this is a more preferable option. For example, here I would look at the operator https://rxjs.dev/api/operators/finalize